Dockerfile

```diff
@@ -257,6 +257,11 @@ COPY --from=gcr.io/projectsigstore/cosign:v1.3.1@sha256:3cd9b3a866579dc2e0cf2fde
 COPY ./Dockerfile.d/test-integration-etc_containerd-stargz-grpc_config.toml /etc/containerd-stargz-grpc/config.toml
 COPY ./Dockerfile.d/test-integration-ipfs-offline.service /usr/local/lib/systemd/system/
 COPY ./Dockerfile.d/test-integration-buildkit-nerdctl-test.service /usr/local/lib/systemd/system/
+COPY ./Dockerfile.d/SHA256SUMS.d/tinit-static-v0.19.0 /SHA256SUMS.d/tinit-static-v0.19.0
+RUN tiniFilename=tini-static-${TARGETARCH:-amd64} && \
+    curl -o "${tiniFilename}" -fSL https://github.com/krallin/tini/releases/download/v0.19.0/tini-static-${TARGETARCH:-amd64} && \
```

Comment:
Use `ARG TINI_VERSION` instead of hardcoded version.

Reply:
Make sense, I will update it later